uint32 iRelay
---
int32 status
